infrastructure/kubernetes/ingress.yml

```diff
@@ -161,12 +168,17 @@ kind: Service
 metadata:
   name: ingress-nginx-service
 spec:
+  # The LoadBalancer resource will be automatically created by Azure, allocating
+  # a public IP address, in case you need a specific IP address you should
+  # create the public IP adddress resource yourself and uncomment the
+  # loadBalancerIP attribute.
   type: LoadBalancer
-  # loadBalancerIP: TODO from static IP via Terraform
+  # loadBalancerIP = output of "azurerm_kubernetes_public_ip_ip" in "test" environment
+  loadBalancerIP: 13.95.82.27
```

Comment:
is this fixed between runs ?

Reply:
yes it's a resource provisioned by Terraform (see `azure.tf`), so until the resource doesn't get deleted we can hold that IP

Reply:
I'll add a line in the readme to check that this IP is in sync with the terraform resource and what to do to check it and eventually to keep them in sync